backend/users/admin.py

```diff
@@ -1,10 +1,31 @@
+import csv
+from io import TextIOWrapper
+
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
+from django.urls import path
+from django.shortcuts import render, redirect
 
-from .forms import CustomUserCreationForm, CustomUserChangeForm
+from .forms import CustomUserCreationForm, CustomUserChangeForm, BulkImportForm
 from .models import CustomUser, Faculty, Student, Department
 
 
+def add_student(row):
+    email = row['email']
+    id_num = row['id number']
+    name = row['name']
+    Student.objects.create(email=email, id_num=id_num, name=name)
+
+def add_faculty(row):
+    email = row['email']
+    name = row['name']
+    psrn = row['psrn']
+    alt_email = row['altername email']
+    contact_num = row['contact number']
+    dept = row['department']
+    Faculty.objects.create(email=email, name=name, psrn=psrn, alt_email=alt_email, contact_num=contact_num, dept=dept)
```

Comment:
What happens when the user already exists (or the same csv is uploaded twice)?

Reply:
User with get an error message that "constraint violated on line number <line number>".

Reply:
![error](https://user-images.githubusercontent.com/31856854/68990809-96134180-087d-11ea-9f3a-1634b66ace90.PNG)

Reply:
Isn't the message very cryptic? Maybe we should specify for *which* user it has failed (ID Num).